docs/deployment.md

```diff
@@ -87,7 +87,11 @@ Enter repository root directory
 
     $ cd <daac>-deploy
 
-Install packages with npm
+Packages are installed with npm. If you're trying to work with a certain version of a cumulus package or task, the version can be specified in `package.json` under dependencies. For example:
+
```

Comment:
I would link to where the packages are on npm. Also note that we use semantic versioning (major/minor/patch) and in your package.json, you can configure for automatic updates:

^ = update minor/patch versions automatically
~ = update patch versions automatically

Reply:
Concurring with @laurenfrederick.

It might be useful to add an example on how to add a package to package.json. For example, `npm install @cumulus/deployment` automatically adds the latest version to the package.json.

Also, we should describe how you can use them in a deployment. This can be achieved by pointing the `source` key of a lambda config to `node_modules/@cumulus/[name-of-the-package]`